PROTO_tests/tests/dtypes_test.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+from arkouda import dtypes
+
+"""
+DtypesTest encapsulates arkouda dtypes module methods
+"""
+
+SUPPORTED_NP_DTYPES = [bool, int, float, str, np.int64, np.float64, np.uint8, np.uint64, np.str_]
+
+
+class TestDTypes:
+    @pytest.mark.parametrize("dtype", SUPPORTED_NP_DTYPES)
+    def test_check_np_dtype(self, dtype):
+        """
+        Tests dtypes.check_np_dtype method
+
+        :return: None
+        :raise: AssertionError if 1.. test cases fail
+        """
+        dtypes.check_np_dtype(np.dtype(dtype))
+
+        with pytest.raises(TypeError):
+            dtypes.check_np_dtype(np.dtype(np.int16))
+        with pytest.raises(TypeError):
+            dtypes.check_np_dtype("np.str")
+        with pytest.raises(TypeError):
+            dtypes.check_np_dtype("bigint")
+
+    @pytest.mark.parametrize(
+        "dtype, byte_size, kind",
+        [
+            (np.bool_, 1, "bool"),
+            (bool, 1, "bool"),
+            (np.int64, 8, "int"),
+            (np.float64, 8, "float"),
+            (np.uint8, 1, "uint"),
+            (np.str_, 0, "str"),
+            (str, 0, "str"),
+        ],
+    )
+    def test_translate_np_dtype(self, dtype, byte_size, kind):
```

Comment:
I think we're getting carried away with parameterization in this file. I don't really see what it's buying us in this case.

It doesn't end up being any less lines of code or easier to read to me. I could be convinced otherwise, i wonder what @Ethan-DeBandi99 thinks. I think if we used the same dictionary in multiple functions it would make sense, but since we only use it once, i'm leaning towards not doing it

Reply:
Is the alternative having 
`d_tuple = dtypes.translate_np_dtype(np.dtype(dtype))
  
assert kind == d_tuple[0]
        assert byte_size == d_tuple[1]`

repeated for each test case (without the parameters, of course)?

Reply:
if i were converting it, i'd do something like the below because I feel like it's more readable
```python
def test_translate_np_dtype(self):
    for b in [np.bool_, bool]:
        assert ("bool", 1) == dtypes.translate_np_dtype(np.dtype(b))

    for s in [np.str_, str]:
        assert ("str", 0) == dtypes.translate_np_dtype(np.dtype(s))

    assert ("int", 8) == dtypes.translate_np_dtype(np.dtype(np.int64))
    assert ("uint", 8) == dtypes.translate_np_dtype(np.dtype(np.uint64))
    assert ("float", 8) == dtypes.translate_np_dtype(np.dtype(np.float64))
    assert ("uint", 1) == dtypes.translate_np_dtype(np.dtype(np.uint8))
```

Reply:
I didn't love the format in the old test file but yes, your suggestion is best of all.  Updated